### ISSUE TITLE
0.4.0 breaks test output

### ISSUE THREAD

**Issue by johnjeffers** [AleG94/mocha-suppress-logs#11]
0.4.0 is suppressing all of the test output. Rolling back to 0.3.1 restored normal behavior.

Output from 0.3.1:
```
npm test

> fusionauth-site-origin-handler@1.0.0 test
> mocha --require mocha-suppress-logs



  test-1
    ✔ /something.html (344ms)

  test-2
    ✔ /docs (123ms)
    ✔ /docs/ (71ms)


  3 passing (544ms)
```

Output from 0.4.0:
```
» npm test

> fusionauth-site-origin-handler@1.0.0 test
> mocha --require mocha-suppress-logs



  test-1

  test-2


  3 passing (533ms)
```

In my test file, I'm using standard `node:assert`
```
import assert from 'node:assert/strict';
```

An example test looks like this:
```
describe('test-1', function() {
  it('/something.html', async function() {
    result = await handler(mockEvent('/something.html'), null)
    assert.deepEqual(result, makeRedirect('/something'));
  });
});
```

```
node --version
v21.1.0
```

Let me know if you need any more info.

**Comment by vassudanagunta** [comment]
Confirmed this is an issue. Looks like the fix for #8 and #9 has an unintended side-affect.

@AleG94 I highly recommend using [@gmrchk/cli-testing-library](https://github.com/gmrchk/cli-testing-library) to add functional tests. You can use it to make assertions on the output of CLI-based execution, e.g. stdout from the execution of mocha tests.

**Comment by AleG94** [comment]
Just released a fix for this with 0.4.1, hoping I'm not missing any particular use case.

Please let me know if everything is working fine for you as well.

Thanks @vassudanagunta for suggesting that awesome package, I did not have time to implement it right now but I will try for the next releases!

**Comment by vassudanagunta** [comment]
Fast turnaround! Confirmed that this has been fixed (and 8 and 9 haven't regressed).

> I did not have time to implement it right now but I will try for the next releases!

Do you want me to submit a PR with a single example test using that package to save you the time of figuring it out?
